src/app/components/Document/index.jsx

```diff
@@ -32,6 +32,7 @@ const Document = ({ assets, app, data, styleTags, helmet }) => {
         {meta}
         <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
         <link rel="manifest" href="/manifest.json" />
+        <GlobalStyle />
```

Comment:
I'd suggest putting the global styles below the resource hints. This is so that the browser can parse the resource hints before getting to any of the styles.